plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -113,6 +120,11 @@ struct SsnData {
   }
 };
 
+/// Custom structure for per transaction data
+struct TxnData {
+  std::string request_body;
+};
+
```

Comment:
Looks like perhaps another plugin that could be simplified by using https://github.com/ywkaras/trafficserver/blob/c53b525f27f10b7cea21c794e2351d6ce4be6e5f/plugins/xdebug/Cleanup.h#L127

Reply:
( an example of use of Cleanup.h is here:  https://github.com/ywkaras/trafficserver/blob/c53b525f27f10b7cea21c794e2351d6ce4be6e5f/plugins/xdebug/xdebug.cc#L62 )